components/CrsSelector.vue

```diff
@@ -24,18 +24,14 @@
 </template>
 
 <script setup>
-  import { useToggle } from "@vueuse/core"
-
   const stepper_tree = inject("stepper_tree")
-  const { geode_object } = stepper_tree
-
   const props = defineProps({
+    input_geode_object: { type: String, required: true },
```

Comment:
Why is it relevant to move the input geode object as a props instead of inside the stepper tree? the input object is directly correlated to the load data and the geode object in the stepper tree